source/_patterns/02-base/02-html-elements/22-list-item/_list-item.scss

```diff
@@ -2,8 +2,15 @@
 // List item styles.
 
 li {
-  @include vertical-spacing(rem(gesso-spacing(1)));
-  margin: 0;
+  margin-bottom: rem(gesso-spacing(1));
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+
+  > *:last-child {
+    margin-bottom: 0;
+  }
```

Comment:
This shouldn’t block the merge.

I know this code was in before we added in the vertical-spacing mixin, but I think we should remove it since we don’t want _all_ elements to have no margin if they’re the last element in a list item.

Example: `<p>` in `<li>` http://localhost:8080/themes/gesso/pattern-lab/?p=base-unordered-list)